spec/mainadditions.html

```diff
@@ -271,6 +271,275 @@ <h1>AvailableNamedTimeZoneIdentifiers ( ): a List of Time Zone Identifier Record
         1. Return _result_.
       </emu-alg>
     </emu-clause>
+
+    <emu-clause id="sec-systemtimezoneidentifier" oldids="sec-defaulttimezone" type="implementation-defined abstract operation">
+      <h1>SystemTimeZoneIdentifier ( ): a String</h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>
+          It returns a String representing the host environment's current time zone, which is either <del>a String representing a UTC offset for which IsTimeZoneOffsetString returns *true*, or a primary time zone identifier</del><ins>a primary time zone identifier or an offset time zone identifier</ins>.
```

Comment:
Why do we need to support " or an offset time zone identifier" for SystemTimeZoneIdentifier?
Currently in ECMA402, the 6.5.3 DefaultTimeZone ( )
https://tc39.es/ecma402/#sup-defaulttimezone
only support "is a valid ([6.5.1](https://tc39.es/ecma402/#sec-isvalidtimezonename)) and canonicalized ([6.5.2](https://tc39.es/ecma402/#sec-canonicalizetimezonename)) time zone name."

which exclude "an offset time zone identifier"
Why do Temporal mandate the engine need to support that for SystemTimeZoneIdentifier? What OS suppor that and what other program in those OS support that? (For example, I try '"TZ="+03:00" date ' on Linux, the Linux 'date' does not support that.